app/workers/organizations/delete_worker.rb

```diff
@@ -0,0 +1,29 @@
+module Organizations
+  class DeleteWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
+
+    # user_id - the user who deletes the organization
+    def perform(organization_id, user_id)
+      org = Organization.find_by(id: organization_id)
+      return unless org
+
+      Organizations::Delete.call(org)
+
+      user = User.find_by(id: user_id)
+      return unless user
```

Comment:
This scares me a bit, we exit the job after the organization has been deleted, if the operator id isn't found or `nil`. Should we exit _before_ ? Otherwise the error handling might still be registering a non existent `user_id`

Reply:
That was clearly an error, thanks for catching! :+1: 
Fixed and added specs for these cases.